### ISSUE TITLE
Virtua Cop 3 : no reflected light

### ISSUE THREAD

**Issue by haejjuk** [Cxbx-Reloaded/Cxbx-Reloaded#2415]
## Quick summary
There is no reflected light on the figures and backgrounds, but there is when running with DXVK

## Details
If you run the old version (ad7328d) with DXVK , you can see reflected light, and later versions don't.
But if i delete CxbxReloaded\HLSL\FixedFunctionVertexShader.hlsl and FixedFunctionVertexShaderState.hlsli
It looks again, but it's too slow.
I'm sorry I used google translator

### System Configuration:
* OS Windows 10 Pro 19041.264
* CPU E3-1240 v3 @ 3.4Ghz
* GPU GeForce GTX 1050 Ti
* etc.

### Additional Information (if any):
![20230302_191212](https://user-images.githubusercontent.com/68559562/222399657-0ab5474d-a280-4118-b4a7-f3309b058603.jpg)
![20230302_191303](https://user-images.githubusercontent.com/68559562/222399843-78469ee4-3fc3-4bdb-8b3d-689f7d582b4d.jpg)
![20230302_191120](https://user-images.githubusercontent.com/68559562/222399962-6479811c-a03c-4a1d-803d-9debd75b1df9.jpg)
![20230302_191005](https://user-images.githubusercontent.com/68559562/222400034-7127bd55-08c6-406b-a1f0-307dc77786b3.jpg)

**Comment by PatrickvL** [comment]
DXVK or any other graphics wrapper (besides recent original GPU vendor drivers) is not supported. The deletion of our HLSL shaders *will* break rendering for most titles, so should not be done. On those grounds, this issue is rightly closed.

However, if indeed a certain rendering seemed to worked correctly in the past and seems broken in the most recent version of CxbxReloaded, while there's no hacking going on, then this might be an acceptable bug report.

Even so. There's a reasonable chance that research into this could conclude the root cause lies in some piece of functionality that improved rendering in many titles, and regressed in only relatively few titles. If that turns out to be the case, the likely outcome would be to keep it unchanged.

So in short, even if someone investigated this, there's no guarantee it'll be fixed.

I'm hesitant to reopen this for above reasons.
